sample/kotlin/src/main/res/layout/fragment_logs.xml

```diff
@@ -60,13 +60,21 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/log_warning" />
 
+    <Spinner
```

Comment:
Isn't there an AppCompatSpinner ?

Reply:
Hmmm...not aware...I will check that out.